latex/architecture.tex

```diff
@@ -475,11 +475,12 @@ \subsection{SYCL Application Memory Model}
 consistent access to the memory from the different devices to the memory
 objects.
 
-To access memory objects from inside a kernel, the user must create an
+To access memory objects, the user must create an
 \gls{accessor} object which parameterizes the type of access
-the kernel requires. 
+that a kernel or the host requires.
```

Comment:
Since you are working on it, the sentence should be clarify about the fact we need 1 accessor per memory object, not only one for all the memory objects.

Reply:
Good point.  I think I've clarified this with the latest commit